### PR TITLE
Unit tests for scenegraph-layer

### PR DESCRIPTION
**Pull request by georgios-uber**

### CODE REVIEW

File: test/modules/layers/scenegraph-layer.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    Layer: ScenegraphLayer,
+    sampleProps: {
+      data: FIXTURES.points,
+      getPosition: d => d.COORDINATES,
+      scenegraph: ''
+    },
+    assert: t.ok,
+    onBeforeUpdate: ({testCase}) => t.comment(testCase.title),
+    onAfterUpdate: ({layer, subLayers}) => {
+      // if (layer.props.mesh) {
```
Comment (ibgreen) [inline test/modules/layers/scenegraph-layer.spec.js]: Nit: Remove comments.

File: test/modules/layers/scenegraph-layer.spec.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+import * as FIXTURES from 'deck.gl-test/data';
+
+test('ScenegraphLayer#tests', t => {
+  const testCases = generateLayerTests({
+    Layer: ScenegraphLayer,
+    sampleProps: {
+      data: FIXTURES.points,
+      getPosition: d => d.COORDINATES,
+      scenegraph: ''
```
Comment (Pessimistress) [inline test/modules/layers/scenegraph-layer.spec.js]: Supply a scenegraph here to test the meaningful part of this layer

Comment (georgios-uber) [reply test/modules/layers/scenegraph-layer.spec.js]: Is it ok if I bundle a glTF file in the repo?

Comment (georgios-uber) [reply test/modules/layers/scenegraph-layer.spec.js]: 📂

File: test/modules/layers/scenegraph-layer.spec.js (hunk 1 above)
Comment (georgios-uber) [reply test/modules/layers/scenegraph-layer.spec.js]: Will do in separate diff.

File: test/modules/layers/scenegraph-layer.spec.js (hunk 2 above)
Comment (georgios-uber) [reply test/modules/layers/scenegraph-layer.spec.js]: Will do in separate diff.

**Pull request merged**